RxExample/Extensions/RxCLLocationManagerDelegateProxy.swift

```diff
@@ -15,14 +15,29 @@ import CoreLocation
 class RxCLLocationManagerDelegateProxy : DelegateProxy
                                        , CLLocationManagerDelegate
                                        , DelegateProxyType {
-    
+
+    internal lazy var didUpdateLocationsSubject = PublishSubject<[CLLocation]>()
+    internal lazy var didFailWithErrorSubject = PublishSubject<Error>()
+
     class func currentDelegateFor(_ object: AnyObject) -> AnyObject? {
         let locationManager: CLLocationManager = object as! CLLocationManager
         return locationManager.delegate
     }
-    
+
     class func setCurrentDelegate(_ delegate: AnyObject?, toObject object: AnyObject) {
         let locationManager: CLLocationManager = object as! CLLocationManager
-        locationManager.delegate = delegate as? CLLocationManagerDelegate
+        if let delegate = delegate {
+            locationManager.delegate = (delegate as! CLLocationManagerDelegate)
+        } else {
+            locationManager.delegate = nil
+        }
+    }
+
+    public func locationManager(_ manager: CLLocationManager, didUpdateLocations locations: [CLLocation]) {
+        didUpdateLocationsSubject.onNext(locations)
```

Comment:
@phlippieb it should do both.
* publish to publish subject -> for observing
* forward it to forwarding delegate -> because of forwarding delegate definition